components/event-chart/src/event-chart.spec.ts

```diff
@@ -194,16 +194,59 @@ describe('DtEventChart', () => {
 
     it('should render the path correctly', () => {
       const path = getRenderedEventsPath(fixture);
-      const expectedPath = [
-        'M13 73',
-        'L146.0666666666667 24',
-        'L212.60000000000002 73',
-        'L345.6666666666667 73',
-        'L478.73333333333335 73',
-        'M678.3333333333334 73',
-        'L1011 24',
-      ].join(' ');
-      expect(path).toBe(expectedPath);
+
+      // Split the path into points and check
+      const splitPath = path
+        .replace(/ ([M,L,C,A])/gim, ' *$1')
```

Comment:
you only work with absolute coordinates?

Reply:
No, that's what the `i` is for, case insensitive.